pkg/adaptation/adaptation.go

```diff
@@ -50,6 +50,7 @@ type SyncFn func(context.Context, SyncCB) error
 type SyncCB func(context.Context, []*PodSandbox, []*Container) ([]*ContainerUpdate, error)
 
 // UpdateFn is a container runtime function for unsolicited container updates.
+// TODO: I think I'll need to add one for PodSandboxUpdates.
```

Comment:
I think UpdatePodSandbox needs to be a purely informative event, not allowing a plugin to request changes to the update in response. Consequently there should not be unsolicited updates requested by a plugin either. Then there is no need for such a new opaque update function either.

Reply:
Acknowledged, removed this

Reply:
can see this going both ways with a switch readOnly/readWriteOverlayRequests